### ISSUE TITLE
Atom v0.199 leaks memory and uses CPU

### ISSUE THREAD

**Issue by pgkos** [atom/atom#6844]
Steps to reproduce (on Ubuntu 14.04, 64-bit, I deleted all configuration files and did a full reinstall of Atom to confirm the issue):

1. Open Atom v0.199.
2. File -> Open folder... -> open e.g. your Linux user's home folder. The folder gets added to the tree view pane.
3. Close Atom.
4. Open Atom again. The previously opened folder shows up again in the tree view pane.
5. Watch the CPU and RAM usage.

On my computer, the whole RAM gets used up after only a minute. Also, the whole first core of my CPU is used while Atom runs. This only happens when there is a folder present in the tree view pane and only after it is restored from memory/history after I restart Atom.

When I remove the folder from the tree view pane, the RAM usage drops and normalizes. Also, the CPU usage drops to 0.

UPDATE: The problem only happens if I open ~/.config folder

**Comment by 50Wliu** [comment]
Hi @pgkos could you see if #6437 covers this issue?  It looks like this is a duplicate of that but I just wanted to make sure.

**Comment by pgkos** [comment]
Hello, yes, it probably is a duplicate of issue #6437. I have noticed that when I disable the fuzzy-finder package the issue disappears.
